pkg/server/events.go

```diff
@@ -144,8 +145,10 @@ func (em *eventMonitor) start() (<-chan struct{}, error) {
 					em.backOff.enBackOff(cID, evt)
 				}
 			case err := <-em.errCh:
-				logrus.WithError(err).Error("Failed to handle event stream")
-				close(closeCh)
+				if err != nil {
```

Comment:
comment here would be nice // if errCH is nil just return (and close errCh) do not report error with event stream

Reply:
Done